spyderlib/widgets/ipython.py

```diff
@@ -293,6 +293,8 @@ def get_signature(self, content):
         data = content.get('data', {})
         text = data.get('text/plain', '')
         if text:
+            if type(text) == type({'e':5}):

```

Comment:
@stonebig, please rewrite this line as

    if isinstance(text, dict):

so I can merge this one :-)